tests/test_security_oauth2_authorization_client_credentials_description.py

```diff
@@ -0,0 +1,77 @@
+from typing import Optional
+
+from fastapi import FastAPI, Security
+from fastapi.security import OAuth2ClientCredentials
+from fastapi.testclient import TestClient
+
+app = FastAPI()
+
+oauth2_scheme = OAuth2ClientCredentials(
+    tokenUrl="token",
+    description="OAuth2 Client Credentials Flow",
+    auto_error=True,
+)
+
+
+@app.get("/items/")
+async def read_items(token: Optional[str] = Security(oauth2_scheme)):
+    return {"token": token}
+
+
+client = TestClient(app)
+
+
+def test_no_token():
+    response = client.get("/items")
+    assert response.status_code == 401, response.text
+    assert response.json() == {"detail": "Not authenticated"}
+
+
+def test_incorrect_token():
+    response = client.get("/items", headers={"Authorization": "Non-existent testtoken"})
+    assert response.status_code == 401, response.text
+    assert response.json() == {"detail": "Not authenticated"}
+
+
+def test_token():
+    response = client.get("/items", headers={"Authorization": "Bearer testtoken"})
```

Comment:
Why are we not setting the `testtoken` in a const file instead of duplicating it everywhere?

Reply:
I followed the existing structure of the other oauth2 tests. I agree `test_token` and possibly the `test_client` should probably be move into fixtures but I think that is beyond the scope of this pull request. I could create a new issue for "reducing code duplication in tests"?